argilla-sdk/docs/guides/how_to_guides/record.md

```diff
@@ -452,4 +450,32 @@ dataset.records.log(records=updated_data)
         updated_records.append(record)
 
     dataset.records.log(records=updated_records)
-    ```
\ No newline at end of file
+    ```
+
+## Delete records
+
+You can delete records in a dataset calling the `delete` method on the `Dataset` object. To delete a record, you need to retrieve them from the server and get a list with those that you want to delete.
+
+```python
+# Delete a single record
+record_to_delete = [list(dataset.records)[0]]
```

Comment:
can't we do `[dataset.records[0]]`? Also, perhaps we don't need to distinguish between delete single or multiple explicilly but just mention we expect a list?

Reply:
No, that's not possible, due to an iterability issue